apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -639,8 +666,30 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                     });
                 });
             } else {
-                metadataProp.options.map((ele: string) => {
-                    allowedList.push({ text: ele, value: ele });
+                metadataProp.options.map((ele: any) => {
+                    if ( !ele.displayName ) {
+                        allowedList.push({ text: ele, value: ele });
+                    } else {
+                        allowedList.push({ content: (
+                            <Table.Row data-testid={ testId }>
```

Comment:
use `data-componentid` instead. Also do not keep it directly as `componentId` use it something like `${componentd}-oidc-table`

Reply:
fixed with [be309bd](https://github.com/wso2/identity-apps/pull/4284/commits/be309bdd61f1ecdcf606cdf0bfe85ea775a210a6)